ui/apps/dashboard/src/components/Insights/InsightsChat/input/SendButton.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button } from '@inngest/components/Button';
+import { RiArrowUpLine } from '@remixicon/react';
+
+type SendButtonProps = {
+  onClick: (e: React.FormEvent) => void;
+  disabled?: boolean;
+};
+
+const SendButton = ({ onClick, disabled }: SendButtonProps) => {
+  return (
+    <Button
+      kind="secondary"
```

Comment:
This should be a primary button instead, and we remove the className entirely, so it uses the default styles for disabled